src/main/java/com/ldtteam/structures/client/BlueprintHandler.java

```diff
@@ -19,12 +20,15 @@
     /**
      * A static instance on the client.
      */
-    private static final BlueprintHandler ourInstance = new BlueprintHandler();
-    private static final int CACHE_SIZE = 30;
-    private static final long CACHE_EVICT_TIME = 45_000L;
+    private static final BlueprintHandler ourInstance      = new BlueprintHandler();
+    private static final int              CACHE_SIZE       = 30;
```

Comment:
Use smaller size for static and something like 15sec (15000ms) evict time. So it doesn't eat so much